### PR TITLE
add cell counts to the platemaps

### PR DESCRIPTION
**Pull request by MikeLippincott**
This PR adds cell counts to the platemap figure for the manuscript

### CODE REVIEW

File: 1.Exploratory_Data_Analysis/scripts/6.heterogeneity_analysis.py [hunk 1]
```diff
@@ -3,12 +3,11 @@
 
 # # This notebook looks into the cell heterogeneity in the control treatments
 
-# In[1]:
+# In[ ]:
```
Comment (axiomcura) [inline 1.Exploratory_Data_Analysis/scripts/6.heterogeneity_analysis.py:6]: Consider rerunning your notebook, all blocks do not have an execution number

File: 1.Exploratory_Data_Analysis/scripts/8.0_create_feature_index.py [hunk 2]
```diff
@@ -1,9 +1,10 @@
 #!/usr/bin/env python
 # coding: utf-8
 
-# In[1]:
+# In[ ]:
```
Comment (axiomcura) [inline 1.Exploratory_Data_Analysis/scripts/8.0_create_feature_index.py:4]: Consider rerunning notebook

File: figures/S1/figures/figureS1.png
Comment (axiomcura) [inline figures/S1/figures/figureS1.png:1]: Consider separating the numbers, as they look squished upon reaching two digits.

File: figures/S1/figures/inhibtor_well_dose_platemap.png
Comment (axiomcura) [inline figures/S1/figures/inhibtor_well_dose_platemap.png:1]: Similar comments about the column numbers

File: figures/S1/scripts/figure_S1.r [hunk 3]
```diff
@@ -349,20 +356,333 @@ ggsave(inhibtor_well_dose_platemap, platemap_plot_inhibitor_dose, width=width, h
...
+
+head(cell_count_df_agg_PBMC)
+
+# Read in the cell count data
+cell_count_df_PBMC <- arrow::read_parquet(cell_count_file_PBMC)
+# aggregate the cell counts oneoneb_Metadata_Treatment_Dose_Inhibitor_Dose with mean and sd
+cell_count_df_well_PBMC <- cell_count_df_PBMC %>%
+  group_by(Metadata_Well, oneb_Metadata_Treatment_Dose_Inhibitor_Dose) %>%
+  summarize(mean = mean(Metadata_number_of_singlecells))
+# rename mean to n
```
Comment (axiomcura) [inline figures/S1/scripts/figure_S1.r:549]: What does "n" mean here?

**Review by axiomcura — APPROVED** [review]
LGTM! Left some comments, feel free to attend those c:

File: figures/S1/figures/figureS1.png
Comment (MikeLippincott) [reply figures/S1/figures/figureS1.png:1]: Addressed!

File: figures/S1/scripts/figure_S1.r (hunk 3 above)
Comment (MikeLippincott) [reply figures/S1/scripts/figure_S1.r:549]: Addressed!

File: figures/S1/figures/inhibtor_well_dose_platemap.png
Comment (MikeLippincott) [reply figures/S1/figures/inhibtor_well_dose_platemap.png:1]: Addressed!

**Pull request merged**